ecl2df/hook_implementations/jobs.py

```diff
@@ -0,0 +1,64 @@
+import importlib
+import os
+from pkg_resources import resource_filename
+
+from ert_shared.plugins.plugin_manager import hook_implementation
+from ert_shared.plugins.plugin_response import plugin_response
+
+
+def _get_jobs_from_directory(directory):
+    resource_directory = resource_filename("ecl2df", directory)
+
+    all_files = [
+        os.path.join(resource_directory, f)
+        for f in os.listdir(resource_directory)
+        if os.path.isfile(os.path.join(resource_directory, f))
+    ]
+    return {os.path.basename(path): path for path in all_files}
+
+
+@hook_implementation
+@plugin_response(plugin_name="ecl2df")
+def installable_jobs():
+    return _get_jobs_from_directory("config_jobs")
+
+
+@hook_implementation
+@plugin_response(plugin_name="ecl2df")
+def installable_workflow_jobs():
+    return {}
```

Comment:
This is not needed if it's not returning anything, but if this is needed in the future I'm fine with leaving it as is!

Reply:
Removed.